### DISCUSSION TITLE
Expand offline capabilities for Analytics

### DISCUSSION THREAD

**Discussion by leo-ori** [invertase/react-native-firebase#6011] [Q&A]
Hello everyone, I've been looking for analytics offline support, I found that events can persist for 72 hrs (after that time those events will be ignored). Is there a way to expand that time? I meant that events can persist longer than 72 hrs

**Comment by mikehardy** [comment]
Hi there! This is unknown at our layer (wrapper around facebook-android-sdk and facebook-ios-sdk) - those two underlying native libraries are in control of this, and...while I don't know for sure I don't recall there being a way to change this. The upstream documentation in those libraries will have the definitive answer, or you'll need to ask on their issues lists
